### DISCUSSION TITLE
[ User permissions / ACL ] Limit Salt channel configuration

### DISCUSSION THREAD

**Discussion by maceoparkore** [uyuni-project/uyuni#6934] [Ideas]
Hi guys,

is it possible to limit user channel configuration. 

i map every salt configuration to system group, after writing sls, i wanna delegate channel configuration modification to some users and limit the users access to specific configuration channels.

The must will be to grant access to system group and by inherit the users could modify the configuration channels associate, i agree it could be not the most use case.

Regards.

**Comment by admd** [comment]
@maceoparkore  RBAC is a big topic and SUSE Manager doesn't have that fine grained permissions capabilities. This is still a topic under radar but nothing has been planned around it.

**Reply by maceoparkore** [reply]
thanks for your reply.
